index.html

```diff
@@ -1,256 +1,276 @@
+                                </ul>
+                            </li>
 
-              <li>
-                <div class="tech_container border_bottom ">
-                  <h6>Frameworks</h6>
-                  <div class="tech_img">
-                    <img src="images/Icon - Chevron - Right - Larger.png" alt="arrow right icon">
-                  </div>
-                </div>
-              </li>
+                            <li>
+                                <div class="tech_container border_bottom">
+                                    <h6>Frameworks</h6>
+                                    <div class="tech_img ">
+                                        <img src="images/Icon - Chevron - Right - Larger.png" alt="arrow right icon ">
+                                    </div>
+                                </div>
+                            </li>
 
-              <li>
-                <div class="tech_container border_bottom ">
-                  <h6>Skills</h6>
-                  <div class="tech_img">
-                    <img src="images/Icon - Chevron - Right - Larger.png" alt="arrow right icon">
-                  </div>
-                </div>
-              </li>
-            </ul>
-          </div>
-        </article>
-      </article>
-    </section>
-  </body>
+                            <li>
+                                <div class="tech_container border_bottom">
+                                    <h6>Skills</h6>
+                                    <div class="tech_img ">
+                                        <img src="images/Icon - Chevron - Right - Larger.png" alt="arrow right icon ">
+                                    </div>
+                                </div>
+                            </li>
+                        </ul>
+                    </div>
+                </article>
+            </article>
+            <article class="contact_me">
+                <H2>Contact me</H2>
+                <p>If you have an application you are interested in developing, a feature that you need built or a
+                    project that needs coding. I'd love to help with it!</p>
+                <form action="https://formspree.io/f/xoqzbvjo" method="post">
+                    <fieldset class="form">
+                        <label for="name">
+                            <input type="text" id="name" maxlength="30" placeholder="Lorem ipsum" required>
+                        </label>
+                        <label for="email">
+                            <input type="email" id="email" placeholder="loremipsum@gmail.com" required>
+                        </label>
+                        <textarea name="message" maxlength="500" id="message" cols="30" rows="10"
+                          placeholder="write your text here" required></textarea>
+                        <br>
+                        <input class="form_btn" type="submit" name="submit" id="submit" value="Submit">
+                    </fieldset>
```

Comment:
- [ ] Kindly make the value of this input `Get in touch` instead of `Submit`. Based on the design, that is the text. Remember that your design should look very identical to that of the Figma template. Kindly implement this change. 👍 😎